src/SFML/Window/CMakeLists.txt

```diff
@@ -242,6 +242,12 @@ if ((NOT ${CMAKE_VERSION} VERSION_LESS 3.11) AND (NOT OpenGL_GL_PREFERENCE))
     set(OpenGL_GL_PREFERENCE "LEGACY")
 endif()
 
+# Compatibility for:
+# - sf::Event::EventType
+if(MML_SFML_COMPAT)
```

Comment:
Let's move it to sfml_add_library macro. We'll already use MML_SFML_COMPAT in sfml-graphics too, so it's better to add this definition to all SFML targets.